### PR TITLE
Use --packages rather than --package-root

### PR DESCRIPTION
**Pull request by abarth**
The .packages file is replacing the packages directory of symlinks.

### CODE REVIEW

**Comment by abarth** [comment]
@devoncarew @johnmccutchan @chinmaygarde

File: packages/flutter_tools/lib/src/package_map.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:package_config/packages_file.dart' show parse;
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/package_map.dart]: perhaps prefix this? `parse` is pretty generic. `... as package_config show parse;`

**Comment by devoncarew** [comment]
lgtm

**Comment by devoncarew** [comment]
Travis has an odd failure.

**Comment by abarth** [comment]
> Travis has an odd failure.

Yeah, I haven't landed the engine part of this patch yet.  It's in https://github.com/flutter/engine/pull/2557

**Pull request merged**